src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/MoreTimeRequestedNotificationService.java

```diff
@@ -30,6 +30,7 @@ public void sendMail(
         final Map<String, String> parameters,
         final String reference
     ) {
+        System.out.println("Sending email to: " + targetEmail);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS106)